packages/web3/src/api/api-explorer.ts

```diff
@@ -2184,4 +2184,4 @@ export class Api<SecurityDataType extends unknown> extends HttpClient<SecurityDa
         ...params
       }).then(convertHttpResponse)
   }
-}
```

Comment:
The diff in this file will fail the CI pipeline.

Reply:
this was a mistake

Reply:
I noticed the changes and tried correcting it back to how it initially was